article/sections/_3_graph_overview.tex

```diff
@@ -0,0 +1,160 @@
+
+\begin{figure}[h!]
+   \centering
+   \includegraphics[height=0.5\textwidth]{simple-graph.pdf}
+   \caption{Simple DAG of the causal relationships between X, Y and Z}
+   \label{fig:simple-graph}
+\end{figure}
+
+In comparison, note the set of structural equations needed to convey the same assumptions as Figure \ref{fig:simple-graph}.
+\[Z = f_Z(X, \epsilon_Z)  \]
+
+\[Y(z) = f_Y(X, z, \epsilon_Y)  \]
+
+Now consider the case where there exists another latent confounding variable, U, which also affects both the treatment Z, as well as the outcome, Y.
+Figure \ref{fig:simple-graph-confounded} illustrates this assumption in DAG form, and the equations below are the structural equation equivalent:
+
+\[Z = f_Z(X, U, \epsilon_Z)  \]
+
+\[Y(z) = f_Y(X, U, z, \epsilon_Y)  \]
+
+\begin{figure}[h!]
+   \centering
+   \includegraphics[height=0.5\textwidth]{simple-graph-confounded.pdf}
+   \caption{Simple DAG of the causal relationships between X, Y, Z and confounder U}
+   \label{fig:simple-graph-confounded}
+\end{figure}
+
+Figure \ref{fig:simple-graph-confounded} shows an indirect connection between Z and Y that goes through U. 
+Therefore, even if we condition on X, omitting U will yield biased results of the relationship between Z and Y, due to the variation in both Z and Y caused by U.
+The structural equations also show that the ignorability assumption of Z does not hold if we only control for X, and we risk obtaining biased estimates of the causal effect of Z on Y if we fail to account for U, sometimes referred to as a common cause.
+Even in this very simple example with only three or four variables, we gain an advantage in parsimony and expressiveness by using a causal graph to encode assumptions.
+The conciseness of DAGs becomes even more useful in real-world applications.
+Here, applied problems may have many available covariates and a complex causal structure.
+In such problems, fully writing out the structural equations for the system may be tedious, and readers may still find it hard to understand.
+
+Another benefit of DAGs is that they come with testable implications.
+Incorporating these tests in any causal analysis adds robustness and defensibility to one's study.
+In section 5, we discuss these implications further, and we illustrate how to tests for them in one's analysis.
+
+Lastly, it is important to note that the graphical approach to causality focuses
+primarily on issues of identification of causal effects. That is, given a
+directed acyclic graph (DAG) that encodes an analyst's knowledge and belief
+about the data generation process of the problem at hand, can a specific
+causal effect be identified?
+As such, we emphasize that DAGs are great tools
+for a modeler to encode their assumptions about a problem, even though we will use more familiar statistical tools to actually infer/estimate a causal effects of interest. 
```

Comment:
I think the compound sentence at "even though" complicates rather than clarifies this sentence. Here's my suggestion!
```suggestion
As such, we emphasize that DAGs are great tools
for a modeler to encode their assumptions about a problem.
We will still use more familiar statistical tools to actually infer/estimate the causal effects of interest. 
```

Reply:
See edits in #176.